### PR TITLE
Scrivito 1.32.0

### PR DESCRIPTION
**Pull request by apepper**
See https://www.scrivito.com/scrivito-js-sdk-1-32-0-released-6dc70e6597de3661 for all the greatness of the release.

### CODE REVIEW

File: src/Widgets/FactWidget/FactWidgetEditingConfig.js [hunk 1]
```diff
@@ -4,6 +4,20 @@ import factWidgetIcon from "../../assets/images/fact_widget.svg";
...
+        toolbar: ["subscript", "superscript"],
+      },
+    },
+    value: {
+      options: {
+        allowedTags: ["sub", "sup"],
+        toolbar: ["subscript", "superscript"],
+      },
+    },
+  },
```
Comment (dcsaszar) [inline src/Widgets/FactWidget/FactWidgetEditingConfig.js:20]: * [ ] Basically same as above. Customers typically ask for sub/sup in headlines. I think `string` is the better default everywhere else. If needed, devs can easily change from string to HTML on their own.

File: src/Widgets/HeadlineWidget/HeadlineWidgetClass.js [hunk 2]
```diff
@@ -2,7 +2,7 @@ import * as Scrivito from "scrivito";
 
 export const HeadlineWidget = Scrivito.provideWidgetClass("HeadlineWidget", {
   attributes: {
-    headline: "string",
+    headline: "html",
```
Comment (dcsaszar) [inline src/Widgets/HeadlineWidget/HeadlineWidgetClass.js:5]: * [ ] If we do this, we must update the `id={speakingUrl(widget.get("headline"))}` part.

**Review by dcsaszar — CHANGES_REQUESTED** [review]
Looks good aside from some rough edges.

See feedback for using the new `allowedTags` feature. I'm not even fully convinced this should be the new default for headlines since it adds a bit of complexity to both the app code and the editing UI. So far, this feature was only requested by large-scale customer projects. Maybe we should just leave the default as is?

**Comment by apepper** [comment]
I forced pushed [798d310](https://github.com/Scrivito/scrivito_example_app_js/pull/518/commits/798d3108d775e9ba0e068a2434464a585c06cd0c) . With that only the upgrade is included

**Pull request merged**